### DISCUSSION TITLE
Converting then importing an N1MM+ ADIF file

### DISCUSSION THREAD

**Discussion by Ciemon** [magicbug/Cloudlog#2234] [General]
I've been using N1MM+ for logging CW contests and would like Cloudlog to display the contacts correctly, so I need to understand what Cloudlog is expecting from the ADIF import file.

For example, during the CWOps CWT the exchange is either CWOps member number or country (or some other letters). The N1MM+ ADIF output looks like this:

<APP_N1MM_EXCHANGE1:4>2381

In this case that 2381 is the other operator's membership number. What do I change the associated reference field to in the ADIF file in order for it to be imported and displayed correctly? 

Thanks you,

73, Ciemon G0TRT

**Comment by phl0** [comment]
I am afraid this is an application specific ADIF field which is not handled in Cloudlog.

**Comment by magicbug** [comment]
As @phl0 says these are tags relating to N1MM so we just ignore them, it's possible that N1MM will include them as the received string in tags is either SRX or SRX_STRING.

**Comment by Ciemon** [comment]
Oh I'm not suggesting that Cloudlog should do anything other than ignore non-standard ADIF fields. I just want to know what field Cloudlog would take the exchange data from in a standard ADIF file; so I'll do some conversion and import tests myself using the SRX_STRING tags. Thank you both.
